OfficeMicroService/Presentation/Controllers/OfficeController.cs

```diff
@@ -0,0 +1,131 @@
+﻿using IdentityMicroService.Domain.Entities.Enums;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OfficeMicroService.Application.Services;
+using OfficeMicroService.Application.Services.DTO;
+using Serilog;
+
+namespace OfficeMicroService.Presentation.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class OfficesController : Controller
+    {
+        private readonly IOfficeServices _officeServices;
+
+        public OfficesController(IOfficeServices officeServices)
+        {
+            _officeServices = officeServices;
+        }
+
+        /// <summary>
+        /// Get All Offices
+        /// </summary>
+        /// <response code="200">Returns offices list</response>
+        /// <response code="500">Operation wasn't succeeded</response>
+        [HttpGet]
+        public async Task<IActionResult> GetAll()
+        {
+            Log.Information("Method {0}", nameof(GetAll));
+            return Ok(await _officeServices.GetAllAsync());
+        }
+
+        /// <summary>
+        /// Get Office Info by id
+        /// </summary>
+        /// <param name="id"></param>
+        /// <response code="200">Returns office info</response>
+        /// <response code="404">Office not found</response>
+        /// <response code="500">Operation wasn't succeeded</response>
+        [HttpGet("{id}")]
+        public async Task<IActionResult> GetById(string id)
+        {
+            Log.Information("Method {0} {1}", nameof(GetById), id);
+            var office = await _officeServices.GetAsync(id);
+            if (office == null)
+            {
+                Log.Information("Method {0}, Office {1} NotFound", nameof(GetById), id);
+                return NotFound();
+            }
+            return Ok(office);
+        }
+
+        /// <summary>
+        /// Create office for Receptionist
+        /// </summary>
+        /// <param name="model"></param>
+        /// <response code="201">Returns created office</response>
+        /// <response code="422">Incorrect model</response>
```

Comment:
Let's move this line down to have status codes written incrementally (201, 401, 403, 422, 500)